README.md

```diff
@@ -4,12 +4,13 @@ A Fabric-based Minecraft modpack focusing on performance and graphics enhancemen
 
 [**📜 Mod list**](INCLUDED-MODS.md) | [**🎛️ Changed options**](https://wiki.download.fo/readme/changed-options) | [**❤️ Support and contribute**](https://download.fo/thanks)
 
-<a href="https://wiki.download.fo/"><img alt="Wiki" height="40" src="https://cdn.jsdelivr.net/npm/@intergrav/devins-badges@3/assets/compact/documentation/gitbook_vector.svg"></a> 
-<a href="https://download.fo/translate"><img alt="Translate" height="40" src="https://cdn.jsdelivr.net/npm/@intergrav/devins-badges@3/assets/compact/translate/generic-plural_vector.svg"></a>
-<a href="https://download.fo/rate"><img alt="Rate" height="40" src="https://github.com/Fabulously-Optimized/fabulously-optimized/assets/8611110/fce9cc22-36a7-4dad-89cf-b47a2f08daa8"></a>
+<a href="https://wiki.download.fo/"><img alt="Wiki" height="50" src="https://cdn.jsdelivr.net/npm/@intergrav/devins-badges@3/assets/compact/documentation/gitbook_vector.svg"></a>&nbsp;&nbsp;&nbsp;

```

Comment:
Checked on desktop, the increased badge size seems too big to me, current size was fine.